setup.py

```diff
@@ -11,11 +11,28 @@
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
-requirements = []
+requirements = [
+    'numpy>=1.14.0',
+    'scipy>=1.5.0',
+    'pyfftw',
```

Comment:
Do we handle pyfftw as a strict requirement or is it optional for improved performance?
from fft.py, line 95:
```try:
    import pyfftw
    pyfftw.config.NUM_THREADS = multiprocessing.cpu_count()
    from pyfftw.interfaces import numpy_fft as fft_lib
except ImportError:
    warnings.warn(
        "Using numpy FFT implementation.\
        Install pyfftw for improved performance.")
    from numpy import fft as fft_lib```

Reply:
Good question. I left it in there, since installing with pip is straight forward. There's a pypi package. It is not distributed via conda though, only conda-forge, that's why I left it as an "optional" dependency.